notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb

```diff
@@ -490,209 +261,31 @@
-      "Indexed 5600 documents\n",
-      "Indexed 5800 documents\n",
-      "Indexed 6000 documents\n",
-      "Indexed 6200 documents\n",
-      "Indexed 6400 documents\n",
-      "Indexed 6600 documents\n",
-      "Indexed 6800 documents\n",
-      "Indexed 7000 documents\n",
-      "Indexed 7200 documents\n",
-      "Indexed 7400 documents\n",
-      "Indexed 7600 documents\n",
-      "Indexed 7800 documents\n",
-      "Indexed 8000 documents\n",
-      "Indexed 8200 documents\n",
-      "Indexed 8400 documents\n",
-      "Indexed 8600 documents\n",
-      "Indexed 8800 documents\n",
-      "Indexed 9000 documents\n",
-      "Indexed 9200 documents\n",
-      "Indexed 9400 documents\n",
-      "Indexed 9600 documents\n",
-      "Indexed 9800 documents\n",
-      "Indexed 10000 documents\n",
-      "Indexed 10200 documents\n",
-      "Indexed 10400 documents\n",
-      "Indexed 10600 documents\n",
-      "Indexed 10800 documents\n",
-      "Indexed 11000 documents\n",
-      "Indexed 11200 documents\n",
-      "Indexed 11400 documents\n",
-      "Indexed 11600 documents\n",
-      "Indexed 11800 documents\n",
-      "Indexed 12000 documents\n",
-      "Indexed 12200 documents\n",
-      "Indexed 12400 documents\n",
-      "Indexed 12600 documents\n",
-      "Indexed 12800 documents\n",
-      "Indexed 13000 documents\n",
-      "Indexed 13200 documents\n",
-      "Indexed 13400 documents\n",
-      "Indexed 13600 documents\n",
-      "Indexed 13800 documents\n",
-      "Indexed 14000 documents\n",
-      "Indexed 14200 documents\n",
-      "Indexed 14400 documents\n",
-      "Indexed 14600 documents\n",
-      "Indexed 14800 documents\n",
-      "Indexed 15000 documents\n",
-      "Indexed 15200 documents\n",
-      "Indexed 15400 documents\n",
-      "Indexed 15600 documents\n",
-      "Indexed 15800 documents\n",
-      "Indexed 16000 documents\n",
-      "Indexed 16200 documents\n",
-      "Indexed 16400 documents\n",
-      "Indexed 16600 documents\n",
-      "Indexed 16800 documents\n",
-      "Indexed 17000 documents\n",
-      "Indexed 17200 documents\n",
-      "Indexed 17400 documents\n",
-      "Indexed 17600 documents\n",
-      "Indexed 17800 documents\n",
-      "Indexed 18000 documents\n",
-      "Indexed 18200 documents\n",
-      "Indexed 18400 documents\n",
-      "Indexed 18600 documents\n",
-      "Indexed 18800 documents\n",
-      "Indexed 19000 documents\n",
-      "Indexed 19200 documents\n",
-      "Indexed 19400 documents\n",
-      "Indexed 19600 documents\n",
-      "Indexed 19717 blogs embeddings documents\n"
-     ]
-    }
-   ],
-   "source": [
-    "BLOGS_FILE=\"data/2023-07-27-blogs-no-embeddings.json\"\n",
-    "df_blogs_embeddings = pd.read_json(BLOGS_FILE, lines=True)\n",
-    "\n",
-    "def gen_rows(df):\n",
-    "  for doc in df.to_dict(orient='records'):\n",
-    "    yield doc\n",
+    "url = \"https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/notebooks/hugging-face/blogs.json\"\n",
```

Comment:
Tested locally and works. 
Colab will work when this Pr is merged (`blogs.json` will exist in  main `elasticsearch-labs` repo).